### DISCUSSION TITLE
getListInputProps should be able to set a value without initialValues

### DISCUSSION THREAD

**Discussion by juliuslissau** [mantinedev/mantine#1840] [Feature requests]
Any chance use-form lists ...form.getListInputProps could be updated, so that it can set values without having an initialValue first?

...form.getInputProps will set a form value without an initialValue, but if using a formList then empty values has to be set for ...form.getListInputProps to work.

<img width="434" alt="one" src="https://user-images.githubusercontent.com/71498286/180597499-f8f387d7-0561-4d89-baee-24c2bdf49baa.png">

<img width="642" alt="two" src="https://user-images.githubusercontent.com/71498286/180597501-430e6e9b-ce74-4bd0-a1cb-ee1c32ef63ad.png">

**Comment by rtivital** [comment]
Form list will be deprecated in 5.0 – https://v5.mantine.dev/form/nested/

**Reply by juliuslissau** [reply]
Ahh awesome, that's much nicer!
Do you know approximately when it will be released?

**Reply by rtivital** [reply]
Next monday

**Reply by juliuslissau** [reply]
Amazing!
